client/src/components/Greeting.js

```diff
@@ -0,0 +1,25 @@
+import "./Greeting.css";
+
+export default function Greeting({ user, isUserLoggedIn }) {
+  const greetings = [
+    "what's cookin'?",
+    "what's happening?",
+    "what's the plan?",
+    "let's get preppin'?",
+    "what's on your remind?",
+    "got a new idea?",
+    "what's lined up?",
+  ];
+
+  const number = Math.floor(Math.random() * greetings.length);

```

Comment:
Cool idea to have different greetings and a simple, clever solution to display them randomly:)